docs/content/en/docs/contribution-guide/development-setup.md

```diff
@@ -130,6 +130,28 @@ If you are getting an error, you can try to run `sudo launchctl load
 /Library/LaunchDaemons/org.virtualbox.startup.plist` (from [a Stackoverflow
 answer](https://stackoverflow.com/questions/18149546/macos-vagrant-up-failed-dev-vboxnetctl-no-such-file-or-directory)).
 
+## Local Development with Apple Silicon Mac
+
+Use [Lima](https://lima-vm.io/) to create a Linux VM if you are using a Mac with
+Apple silicon. For example:
+
+```shell
+brew install lima
+limactl create --tty=false --name=tetragon \
+    --set='.vmType = "vz" | .nestedVirtualization = true | .mounts[] |= (select(.location == "~") | .writable = true)'
```

Comment:
nit: I think [vz might be the default now with lima 1.0](https://github.com/lima-vm/lima/pull/1951), and that nestedVirt is not strictly needed in our situation. Also not sure if it's a good idea to recommend mounting the whole home folder as writable, I think they made it read only in the past because of potential issues.

Reply:
cool i'll update it.

i guess there is a way to only make the tetragon directory writable? 💭

Reply:
let's start with this ☕ 

<img width="857" alt="Screenshot 2024-12-17 at 11 14 55 AM" src="https://github.com/user-attachments/assets/d708b9da-286d-4af4-bc1f-6efa7da36085" />